utils/ccloud_library.sh

```diff
@@ -724,7 +722,7 @@ END
   )
   echo "$OUTPUT"
   if [[ ! "$OUTPUT" =~ "Logged in as" ]]; then
-    echo "Failed to log into your cluster.  Please check all parameters and run again"
+    echo "Failed to log into your cluster. Please check all parameters and run again."
```

Comment:
The regex in 747 needs to change as well to match the new service account id format:

```
  if ! [[ "$serviceAccount" =~ ^sa[-][a-z0-9]+$ ]]; then
```

Reply:
Why do we need `[]` around the minus sign?

Reply:
I think we're fine without